src/helpers/cards.ts

```diff
@@ -763,4 +766,83 @@ export const updateCardSettings = async (
   return settings;
 };
 
+export const createCardSpendingLimit = async (
+  req,
+  res
+): Promise<CardSpendingLimitControl> => {
+  const { scope_id: cardId, limit, idempotency_key: idempotencyKey } = req.body;
```

Comment:
Don't we need `scope` param in body as defined in doc ?
![Screen Shot 2023-02-17 at 17 25 41](https://user-images.githubusercontent.com/39880954/219681492-5e4e81d4-6fe5-4b4f-b729-101effe2ce90.png)

Reply:
we will use only CARD scope and there is no need to add more
we always add only the necessary logic 
I can return [] if the scope is not CARD